### DISCUSSION TITLE
ChatGPT

### DISCUSSION THREAD

**Discussion by webb8020** [jimmejardine/qiqqa-open-source#405] [Ideas]
Have you thought about integrating the Quiqqa library with ChatGPT? It could be a game-changer for scientific modeling and research. With access to peer review journal articles and books our libraries, how much faster could we find and analyze relevant information? Plus, the chatbot could filter through and find the most relevant articles more efficiently.

**Comment by wwweagle** [comment]
Is it your intention develop a tool-chain to train and deploy a GPT3.5-level model locally using a personalized scientific article library? It is not impossible, but you will need a massive library (to the point of being unintelligible for humans) and a big budget for computational power. And years of experience for interpreting and fine-tuning the result. I guess it might be a interesting concept for a PhD thesis.

**Reply by webb8020** [reply]
Yes

**Reply by wwweagle** [reply]
Oh. Respect. Best wishes, and please keep us informed. The outcome may well be game-changing indeed.

**Comment by TawandaVera** [comment]
I was thinking the same, especially if we can get the gpt 3.5 api to work with the expedition and brainstorm, it will make the themes and auto tagging into a super power. I see Atlas ti is exploring this.
